documentation/IDTA-01003-a/modules/ROOT/pages/IDTA-01003-a_ChangeLog.adoc

```diff
@@ -1,3 +1,13 @@
+////
+Copyright (c) 2023 Industrial Digital Twin Association
```

Comment:
Shouldn't it be "Copyright (c) 2024" ?

Reply:
probably 
```
Copyright (c) 2023-2024 Industrial Digital Twin Association
Copyright (c) 2018-2022 Zentralverband der Elektro- und Digitalindustrie and Plattform Industrie 4.0
```
see e.g. https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work#:~:text=In%20the%20case%20of%20open%20source%20projects%2C%20the,commit%20to%20a%20publicly%20visible%20version%20control%20repository.

The year in the copyright notice refers to the year of first publication. 
It is a common convention that the notice lists all years where new versions were published.

Since this is relevant for ALL files in all aas-specs repositories I will check this with the Open Source Office.

Reply:
I added an additional issue for this so that we can close this PR, ok?

Reply:
Thanks, that is a good suggestion.